### ISSUE TITLE
[BUG] mars scheduler exit with exception

### ISSUE THREAD

**Issue by sandflee** [mars-project/mars#157]
```
Unhandled exception in promise call
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/mars/promise.py", line 294, in _wrapped
    return func(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/mars/utils.py", line 287, in _wrapped
    return func(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/mars/scheduler/assigner.py", line 288, in _allocate_resource
    input_sizes = dict((k, meta.chunk_size) for k, meta in metas.items())
  File "/usr/lib64/python2.7/site-packages/mars/scheduler/assigner.py", line 288, in <genexpr>
    input_sizes = dict((k, meta.chunk_size) for k, meta in metas.items())
AttributeError: 'NoneType' object has no attribute 'chunk_size'
```

**Comment by wjsi** [comment]
@sandflee  Please provide your Mars version and the way to reproduce this error according to the issue template.

**Comment by sandflee** [comment]
mars version is 0.1.0b1.

scheduler: bjlt-h2571.sy
mars-scheduler -a bjlt-h2571.sy -p 12001 --level DEBUG

web: bjlt-h2571.sy
mars-web -a bjlt-h2571.sy -s bjlt-h2571.sy:12001 --ui-port 12002

worker:[host1,host2,host3]
mars-worker -a 10.48.132.35  -p 12003 -s bjlt-h2571.sy:12001 --cache-mem 51200m --phy-mem 90%

just run distributed demo:
import mars.tensor as mt
from mars.session import new_session
sess = new_session('http://10.48.91.27:12002')
a = mt.ones((2000, 2000), chunk_size=200)
b = mt.inner(a, a)
print sess.run(b)

**Comment by wjsi** [comment]
We have some known issues in v0.1.0b1 that prevents distributed environment from returning correct results. These bugs will be solved in the coming release with in this week. You can also install from v0.1 branch to get a patched version by 

```bash
pip install git+https://github.com/mars-project/mars.git@v0.1
```

What's more, it is recommended to run Mars with Python 3. We don't optimize our code with Python 2 and will drop support on it someday in the future.

**Comment by sandflee** [comment]
thank u! @wjsi

**Comment by wjsi** [comment]
Fix #130 released with v0.1.0.
